tests/integration/conftest.py

```diff
@@ -1288,3 +1300,13 @@ def truncate_event_logs():
 
     for log_file in log_files:
         truncate_file(log_file)
+
+
+def pytest_deselected(items):
```

Comment:
Please, add some documentation here.

Reply:
I know this is a pytest hook, but why are you using it and what does it do?

Reply:
It was a debugging fixture planned to help the develop of the new marker. Good catch. 
Removed here: https://github.com/wazuh/wazuh-qa/pull/3972/commits/22ff92e62412cca92138dab40d2f5a21b7bf0267